examples/env_check.yaml

```diff
@@ -24,7 +24,27 @@ run: |
     exit 1
   fi
 
-  echo NODE ID: $SKY_NODE_RANK
-  echo NODE IPS: "$SKY_NODE_IPS"
+  if [[ -z "$SKY_NODE_RANK" ]]; then
+    echo "SKY_NODE_RANK is not set"
+    exit 1
+  else
+    echo "SKY_NODE_RANK is set to ${SKY_NODE_RANK}"
+  fi
+
+  if [[ -z "$SKY_NODE_IPS" ]]; then
+    echo "SKY_NODE_IPS is not set"
+    exit 1
+  else
+    echo "SKY_NODE_IPS is set to ${SKY_NODE_IPS}"
+    echo "${SKY_NODE_IPS}"
+    echo "${SKY_NODE_IPS}" | wc -l | grep 2 || exit 1
+  fi
   worker_addr=`echo "$SKY_NODE_IPS" | sed -n 2p`
   echo Worker IP: $worker_addr
+
+  if [[ -z "$SKYPILOT_RUN_ID" ]]; then
+    echo "SKYPILOT_RUN_ID is not set"
```

Comment:
Why is it `SKYPILOT_RUN_ID` instead of `SKY_RUN_ID`?

Reply:
This is good point! I think the main reason is that we were trying to gradually move to `SKYPILOT` for the environment variables. Just raise a vote.

Reply:
It seems we decided to use both `SKY` and `SKYPILOT` as our env var prefixes. Would you change the env var names accordingly?

Reply:
I will. Changing it right now. Was waiting for your vote. ; )